phpBB/styles/prosilver/theme/images/contact-icons.html

```diff
@@ -0,0 +1,65 @@
+		</g>
+	</symbol>
+	<symbol id="jabber" viewBox="0 0 24 24">
+		<g fill-rule="evenodd">
+			<path fill="#CC0000" d="M12.572 20.49c-.315-1.514-.316-2.717.464-4.143.116-.214 1.268-1.065 1.537-1.277 1.311-1.034 3.084-2.797 3.158-4.55-.006-2.75-1.314-5.145-3.936-6.352-1.125-.518-1.92-.66-3.118-.832-1.784.175-3.262.432-4.425 1.89a.718.718 0 0 1-.985.115.669.669 0 0 1-.117-.956c.664-.8 1.177-1.179 2.11-1.654.714-.365 1.733-.592 2.53-.682 1.078-.12 1.843-.015 2.895.269 1.995.54 3.79 1.704 4.95 3.38.345.497.542.936.795 1.473.235.497.316 1.046.446 1.57.125.505.151 1.41.098 1.933-.284 2.736-3.027 4.713-5.153 6.197-.532.988-.902 2.371-.554 3.473a.346.346 0 0 1-.272.41.357.357 0 0 1-.423-.265z"
+			/>
+			<path fill="#CC0000" d="M10.076 14.504c-.044-.195.016-.386.126-.56.169-.269.456-.483.668-.73.24-.288.416-.604.586-.925.127-.248.33-.451.127-.71a.552.552 0 0 0-.11-.112c-.168-.124-.445-.005-.611.068-.356.157-.674.383-.983.595-.096.063-.276.24-.403.25-.053.003-.08-.06-.097-.091a1.537 1.537 0 0 1-.134-.335c-.098-.337-.06-.713-.217-1.036a.478.478 0 0 0-.596-.22c-.274.097-.427.35-.502.576-.185.53.004 1.047.036 1.583.013.228.11.765-.282.82-.131.018-.344-.008-.434-.101-.049-.054-.059-.147-.081-.21a5.562 5.562 0 0 1-.15-.538c-.135-.565-.225-1.206-.03-1.769.099-.282.272-.558.582-.716.622-.314 1.51-.154 1.927.34.101.124.17.267.213.415l.057.449.454-.415c.24-.217.514-.408.831-.537a1.84 1.84 0 0 1 .382-.102c.106-.014.225-.023.331-.007.561.084.736.692.736 1.092.002.487-.138.971-.426 1.403-.24.354-.574.661-.86.995-.148.17-.286.345-.435.516-.081.096-.18.2-.325.228-.194.036-.343-.06-.38-.216z"
+			/>
+			<path fill="#CC0000" d="M6.265 17.32c-.359-.256-.718-.578-.812-1.005-.055-.25-.005-.478.193-.597l.244-.082.252.016c.224.067.372.264.532.447.345.4.711.798 1.162 1.1.562.37 1.293.515 1.968.412.228-.033.447-.094.668-.152.125-.032.271-.085.403-.066.045.008.1.019.14.04.303.146.07.49-.152.583-.411.178-.902.247-1.351.265a5.311 5.311 0 0 1-3.247-.961zm3.289 2.888a6.885 6.885 0 0 1-3.37-1.118c-.362-.237-.723-.506-.997-.83-.095-.11-.294-.357-.112-.48.086-.058.201-.055.301-.066h-.001a.908.908 0 0 1 .334.006c.196.062.352.267.503.39.25.204.518.387.81.537a5.885 5.885 0 0 0 2.49.643c.324.008.647.003.97-.038.135-.017.343-.092.474-.035.296.128.394.712.06.858-.118.05-.278.06-.404.08a5.064 5.064 0 0 1-1.058.053zM7.143 21.59a7.133 7.133 0 0 1-1.353-.585c-.379-.225-.927-.67-.723-1.143.039-.085.1-.147.165-.214.095-.1.201-.17.347-.184v-.001c.147-.016.286.093.383.183.224.208.382.464.592.683.063.067.157.112.235.163.262.172.56.297.859.4.35.121.717.205 1.08.29.422.098.819.187 1.259.138a4.12 4.12 0 0 0 .969-.243c.132-.05.309-.174.455-.125.245.083.154.278.02.407-.239.23-.563.4-.889.493-1.1.311-2.342.087-3.4-.262z"
+			/>
+		</g>
+	</symbol>
+	<symbol id="pm" viewBox="0 0 24 24">
```

Comment:
Any reason why half of the icons have a phpBB prefix and the others don't? I'd rather use prefixes for all contact icons than not mixing this or not having a prefix. Especially since IDs are somewhat invasive in their nature of wanting to be unique.